src/Constant.mts

```diff
@@ -4,11 +4,11 @@
  */
 
 import { Constants } from 'discord.js'
-import { toUpperCase } from 'io/github/sayakie/hakase/util/ArrayUtil.mjs'
-import { createEnum } from 'io/github/sayakie/hakase/util/function.mjs'
 import { dirname, join, resolve } from 'node:path'
 import { fileURLToPath } from 'node:url'
 
+import { ArrayUtil, Util } from './index.mjs'
```

Comment:
**Util** throws a ReferenceError, which messaged `"Cannot access 'Util' before initialization"`.

How about apply like this:
```suggestion
import { ArrayUtil} from './index.mjs'
import { createEnum } from './util/function.mjs'
```